onyo/lib/tests/test_commands_mv.py

```diff
@@ -36,14 +36,6 @@ def test_onyo_mv_errors(inventory: Inventory) -> None:
                   destination=dir_path / "doesnotexist" / "somewhere",
                   message="some subject\n\nAnd a body")
 
-    # move directory to existing file
-    pytest.raises(ValueError,
```

Comment:
Instead, this should be a test that checks for an error if `onyo mv` is tried to be used to rename an asset dir, which is illegal.

Related to #562

Reply:
Done.